src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.ui.xml

```diff
@@ -56,6 +55,20 @@
             <span ui:field="spanPatternExample_" class="{style.example}">Example: *.R, *.r, *.csv. Separate multiple types with commas.</span>
          </div>
       </p>
+
+      <p><rw:FormLabel ui:field="labelExcludeFilePatterns_" text="Exclude these files:"/>
+         <g:ListBox ui:field="listPresetExcludeFilePatterns_"
+                    styleName="{style.presetFilePatterns}">
+            <g:item value="">None</g:item>
+            <g:item value="gitExclusions">Standard Git exclusions</g:item>
```

Comment:
Because we now require `git` to make this work, selecting this option is always going to result in an error if (a) git is not installed, or (b) the directory to search is not a git repo. How do you feel about hiding or disabling it up front in these cases? (Both are pretty easy to deduce, and (a) is part of SessionInfo so would be an easy change)

Reply:
I like the idea of enabling or disabling the option up front and locally implemented the check for if git is installed. Is it possible to determine if a directory is a git repo without a server call? 
Right now I just have it implemented so the back end will alert the user when it starts processing the find request.

Reply:
I added a isGitDirectory RPC to Files Server and call this when the directory changes to check if it's a git repo or not - then enable/disable the git option accordingly. This is ready for another review.